src/main/java/com/adobe/aem/compgenerator/utils/DialogUtils.java

```diff
@@ -298,6 +348,45 @@ private static Node updateDefaultNodeStructure(Document document, Element root)
                 .appendChild(createUnStructuredNode(document, "items"));
     }
 
+    /**
+     * Creates the default tab node structure.
+     *
+     * @param document The {@link Document} object
+     * @param root The {@link Element} object
+     * @return the node
+     */
+    private static Node createDefaultTabNodeStructure(Document document, Element root) {
```

Comment:
Is this more appropriately named `createTabsContainerNodeStructure` or `createTabsParentNodeStructure`?  When I see "default" I think it's a generic form of `createTabStructure` but that's not what this is doing.

Reply:
Agreed, updated the method name